src/pcapi/routes/native/v1/favorites.py

```diff
@@ -96,6 +98,9 @@ def get_favorites(user: User) -> serializers.PaginatedFavoritesResponse:
 @spectree_serialize(response_model=serializers.FavoriteResponse, on_error_statuses=[400], api=blueprint.api)  # type: ignore
 @authenticated_user_required
 def create_favorite(user: User, body: serializers.FavoriteRequest) -> serializers.FavoriteResponse:
+    if settings.MAX_FAVORITES:
+        if Favorite.query.filter(Favorite.user == user).count() >= settings.MAX_FAVORITES:
```

Comment:
micro-détail : 

```suggestion
        if Favorite.query.filter_by(user=user).count() >= settings.MAX_FAVORITES:
```

(j'ai un peu de mal avec la lourdeur de `filter()`, donc quand on peut l'éviter. ;)

Reply:
Le problème avec les warnings pylint ?
d'ailleurs, est-ce que l'on a moyen de les activer en local pour éviter d'amplifier le problème ? Je ne sais pas trop quelles sont les règles problématiques.

Reply:
> Le problème avec les warnings pylint ?

Désolé, je n'ai pas été explicite. Mon commentaire n'a pas à voir avec pylint. C'est juste que je trouve que `filter(Favorite.user == user)` est moins lisible (plus verbeux) que `filter_by(user=user)`.

> d'ailleurs, est-ce que l'on a moyen de les activer en local pour éviter d'amplifier le problème ?

Je pense que tu peux appeler pylint comme ça : `pylint --enable singleton-comparison`.

Mais ce message n'est levé que par `filter(Model.value == True)`. pylint ne se plaint pas de `filter(Model.value == user)`.